code/game_of_life/game_of_life.py

```diff
@@ -0,0 +1,119 @@
+class GameOfLife(object):
+    def start(self, string_field):
+        if string_field == "":
+            return ""
+        parser = GameOfLifeParser()
+        field = parser.parse(string_field)
+        field_boundaries = self.calc_field_boundaries(field)
+        next_field = self.step_calculation(field, field_boundaries)
+        res_string = self.convert_to_string(next_field)
+        return res_string
+
+    def calc_field_boundaries(self, current_field):
+        field_boundaries = {}
+        height_field = len(current_field)
+        width_field = len(current_field[0])
+        field_boundaries["height_field"] = height_field
+        field_boundaries["width_field"] = width_field
+        return field_boundaries
+
+    def create_empty_field(self, field_boundaries):
+        next_field = [[0 for i in range(field_boundaries["width_field"])]
+                      for j in range(field_boundaries["height_field"])]
+        return next_field
+
+    def step_calculation(self, current_field, field_boundaries):
+        next_field = self.create_empty_field(field_boundaries)
+        for i in range(field_boundaries["height_field"]):
+            for j in range(field_boundaries["width_field"]):
+                current_neighbors = self.counting_neighbors(current_field, i, j)
+                if current_field[i][j]:
```

Comment:
I think `if current_field[i][j] == 1` looks better. It works in your task, but I think it's better to do this when you have the bool variable.